ui/app/templates/storage/volumes/dynamic-host-volume.hbs

```diff
@@ -0,0 +1,115 @@
+        {{this.model.pluginID}}
+      </span>
+      <span class="pair" data-test-volume-create-time>
+        <span class="term">Create Time</span>
+        <span class="tooltip" aria-label="{{format-month-ts this.model.createTime}}">
+          {{moment-from-now this.model.createTime}}
+        </span>
+      </span>
+      <span class="pair" data-test-volume-modify-time>
+        <span class="term">Modify Time</span>
+        <span class="tooltip" aria-label="{{format-month-ts this.model.modifyTime}}">
+          {{moment-from-now this.model.modifyTime}}
+        </span>
+      </span>
+      {{#if this.model.capacityBytes}}
+        <span class="pair" data-test-volume-capacity>
+          <span class="term">Capacity</span>
+          {{format-bytes this.model.capacityBytes}}
+        </span>
+      {{/if}}
+    </div>
+  </div>
+
+  <div class="boxed-section">
+    <div class="boxed-section-head">
+      Allocations
+    </div>
+    <div class="boxed-section-body {{if this.model.allocations.length "is-full-bleed"}}">
+      {{#if this.sortedAllocations.length}}
+        <ListTable
+          @source={{this.sortedAllocations}}
+          @class="with-foot" as |t|>
+          <t.head>
+            <th class="is-narrow"><span class="visually-hidden">Driver Health, Scheduling, and Preemption</span></th>
+            <th>ID</th>
+            <th>Created</th>
+            <th>Modified</th>
+            <th>Status</th>
+            <th>Client</th>
+            <th>Job</th>
+            <th>Version</th>
+            <th>CPU</th>
+            <th>Memory</th>
+          </t.head>
+          <t.body as |row|>
+            <AllocationRow
+              {{keyboard-shortcut
+                enumerated=true
+                action=(action "gotoAllocation" row.model)
+              }}
+              @data-test-allocation={{row.model.id}}
+              @allocation={{row.model}}
+              @context="volume"
+              @onClick={{action "gotoAllocation" row.model}} />
+          </t.body>
+        </ListTable>
+      {{else}}
+        <div class="empty-message" data-test-empty-allocations>
+          <h3 class="empty-message-headline" data-test-empty-allocations-headline>No Allocations</h3>
+          <p class="empty-message-body" data-test-empty-allocations-message>No allocations are making use of this volume.</p>
+        </div>
+      {{/if}}
+    </div>
+  </div>
+
+  <div class="boxed-section">
+    <div class="boxed-section-head">
+      Capabilities
+    </div>
+    <div class="boxed-section-body is-full-bleed">
+      <table class="table">
+        <thead>
+          <th>Setting</th>
+          <th>Value</th>
+        </thead>
+        <tbody>
+          {{#each this.model.capabilities as |capability|}}
+            <tr data-test-capability-row>
+              <td data-test-capability-key>{{capability.key}}</td>
+              <td data-test-capability-value>{{capability.value}}</td>
+            </tr>
+          {{/each}}
```

Comment:
This is iterating over capabilities but creating a row for each key-value pair. Which means the pairing of access mode and attachment mode that happens in ex. https://github.com/hashicorp/nomad/blob/main/enos/modules/run_workloads/scripts/nfs-volume.hcl#L9-L22 isn't being preserved.

That is, we want the table to look more like this:

**Capabilities**
| Access Mode  | Attachment Mode |
|--|--|
| single-node-writer | file-system |
| single-node-reader| file-system|
| single-node-single-writer|block|